README.md

```diff
@@ -11,6 +16,16 @@
 
 </div>
 
+> [!Caution]
+> This README content has not been validated recently.
+>
+> Glazier is no longer supported, and should not be used.
+>
+> This repository is made available for reference purposes.
+>
+> Feel free to use the [#glazier channel](https://xi.zulipchat.com/#narrow/stream/351333-glazier) if reusing code from Glazier.
+> Some of the text input handling code is better supported than the equivalent in `winit`.
```

Comment:
Feels like "supported" is the wrong word here given "Glazier is no longer supported" ...